Signal/translations/en.lproj/Localizable.strings

```diff
@@ -472,9 +475,15 @@
 "MSGVIEW_RECEIVED_CALL" = "You received a call from %@.";
 
 /* No comment provided by engineer. */
+"MSGVIEW_THEY_TRIED_TO_CALL_YOU" = "%@ tried to call you.";
```

Comment:
Interestingly, this message is only seen if (after future changes) you leave the call view and enter the conversation view _while receiving an incoming call_.  I went for the past tense to be consistent, and since these messages represent "events", but it might be more accurate to say "Alice is trying to call you."